src/background/subscribe.ts

```diff
@@ -15,7 +15,7 @@ export function subscribe(store: Store<AppState, Action>, browser: typeof window
     previousState = nextState
 
     // Dispatch actions when the extension was clicked
-    if (nextState.popupConnected && !prevState.popupConnected) {
+    if (nextState.popupConnected && !prevState.popupConnected && !nextState.tweeting) {
```

Comment:
Wouldn't this prevent the users to use the extension as long as one tab is tweeting? What happens if the user has a slow connection and it takes time to tweet? Shouldn't we check if we are tweeting from the active tab?